docs/upgrade.html

```diff
@@ -1699,7 +1699,7 @@ <h4><a id="upgrade_2_0_0" href="#upgrade_2_0_0">Upgrading from 0.8.x, 0.9.x, 0.1
 <h5><a id="upgrade_200_notable" href="#upgrade_200_notable">Notable changes in 2.0.0</a></h5>
 <ul>
     <li><a href="https://cwiki.apache.org/confluence/x/oYtjB">KIP-186</a> increases the default offset retention time from 1 day to 7 days. This makes it less likely to "lose" offsets in an application that commits infrequently. It also increases the active set of offsets and therefore can increase memory usage on the broker. Note that the console consumer currently enables offset commit by default and can be the source of a large number of offsets which this change will now preserve for 7 days instead of 1. You can preserve the existing behavior by setting the broker config <code>offsets.retention.minutes</code> to 1440.</li>
-    <li>Support for Java 7 has been dropped, Java 8 is now the minimum version required.</li>
+    <li>Support for Java 7 and Java 8 has been dropped, Java 11 is now the minimum version required.</li>
```

Comment:
why you add change to 2.0.0 release? we should update 4.0.0 release, right?

Reply:
Yes, I will revert this change.